From 6eb2fe0724bd235f6f83d28c8f9c22666854bf2a Mon Sep 17 00:00:00 2001
From: Qu Wenruo <wqu@suse.com>
Date: Fri, 19 Jun 2026 18:24:46 +0930
Subject: [PATCH 11974/15333] btrfs: remove out-of-date comments regarding 2K
 block size

Since commit bac3c2910c0c ("btrfs: remove 2K block size support") there
is no 2K block size support inside btrfs anymore.

Remove the stale comments of btrfs_supported_blocksize().

Reviewed-by: Johannes Thumshirn <johannes.thumshirn@wdc.com>
Signed-off-by: Qu Wenruo <wqu@suse.com>
Reviewed-by: David Sterba <dsterba@suse.com>
Signed-off-by: David Sterba <dsterba@suse.com>
---
 fs/btrfs/fs.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/fs/btrfs/fs.c b/fs/btrfs/fs.c
index dcf12979af33..5bd6856213aa 100644
--- a/fs/btrfs/fs.c
+++ b/fs/btrfs/fs.c
@@ -135,12 +135,6 @@ void btrfs_csum_final(struct btrfs_csum_ctx *ctx, u8 *out)
  *
  * - PAGE_SIZE
  *   The straightforward block size to support.
- *
- * And extra support for the following block sizes based on the kernel config:
- *
- * - MIN_BLOCKSIZE
- *   This is either 4K (regular builds) or 2K (debug builds)
- *   This allows testing subpage routines on x86_64.
  */
 bool __attribute_const__ btrfs_supported_blocksize(u32 blocksize)
 {
-- 
2.55.0

